src/tests.js

```diff
@@ -64,11 +54,18 @@ function Test(component, config) {
     }
   }
 
+  const mixins = config.mixins === undefined ? {} : config.mixins
+
   return testComponent.mixin({
     find: Find,
     setState: SetState,
-    simulate: Simulate
+    simulate: Simulate,
+    ...mixins
   })
 }
 
+export const addMixins = (Test, mixins) => {
+  return (component, config) => Test(component, { ...config, mixins })
+}
```

Comment:
It would be nicer to do CustomTest = addMixins(mixins) so that you wouldnt need to require Test and pass it in, the function should do it for you